modules/candidate_profile/jsx/CandidateInfo.js

```diff
@@ -160,7 +160,7 @@ export class CandidateInfo extends Component {
             };
             return (
                 <div style={cardStyle} key={info.label}>
-                    <dt>{info.label}</dt>
+                    <dt style={{whiteSpace: 'nowrap'}}>{info.label}</dt>
```

Comment:
would need it for the info.value too

Reply:
no, we want info.Value to wrap

Reply:
oh, I see what you mean in the screenshot. I'll add it to the DoB value tomorrow. We don't want that one to wrap.

Reply:
It's also a nice change for the headers to not wrap though